src/_data/data.js

```diff
@@ -59,7 +68,8 @@ function parseMarkdown(rawContent, filename, category) {
     category,
     content: parsed.content,
     frontmatter: parsed.data,
-    question,
+    question: questionRaw,        // Keep markdown formatting for display
+    questionPlain: questionPlain,  // Plain text for page titles
```

Comment:
`questionTxt`? `title` ? Or something that makes it a little clearer which is which?